src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -143,36 +138,26 @@ public static object GetUninitializedObjectWithFormatterServices(Type type)
 
         #region Static initialization
 
-        public static void InitializeForTesting(List<TypeInfo> serializationProviders = null, bool useJsonFallbackSerializer = false)
+        public static void InitializeForTesting(List<TypeInfo> serializationProviders = null)
         {
             try
             {
                 RegisterBuiltInSerializers();
                 BufferPool.InitGlobalBufferPool(new MessagingConfiguration(false));
                 RegisterSerializationProviders(serializationProviders);
                 AssemblyProcessor.Initialize();
-                fallbackSerializer = GetFallbackSerializer(useJsonFallbackSerializer);
+                fallbackSerializer = GetFallbackSerializer();
             }
             catch (ReflectionTypeLoadException ex)
             {
                 throw ex.Flatten();
             }
         }
 
-        internal static void Initialize(List<TypeInfo> serializationProviders, bool useJsonFallbackSerializer)
+        internal static void Initialize(List<TypeInfo> serializationProviders)
         {
             RegisterBuiltInSerializers();
-
-#if NETSTANDARD
```

Comment:
Oh, I misread the diff. I'm not sure that we need a warning which the user cannot do anything about. On the other hand, in `GetFallbackSerializer()`, I'm throwing unconditionally on `NETSTANDARD`, which is clearly untenable. What's the other option, though, other than to wait for #2162 for `NETSTANDARD` support.

We could return a fallback serializer which always throws.